src/main/scala/org/broadinstitute/dsde/firecloud/model/Profile.scala

```diff
@@ -52,10 +77,20 @@ object Profile {
       fckv:FireCloudKeyValue => (fckv.key.get -> fckv.value.get) }).toMap
 
     new Profile(
-      name = mappedKVPs.get("name").get,
-      email = mappedKVPs.get("email").get,
-      institution = mappedKVPs.get("institution").get,
+      firstName = mappedKVPs.get("firstName").get,
+      lastName = mappedKVPs.get("lastName").get,
+      title = mappedKVPs.get("title").get,
+      institute = mappedKVPs.get("institute").get,
+      institutionalProgram = mappedKVPs.get("institutionalProgram").get,
+      programLocationCity = mappedKVPs.get("programLocationCity").get,
+      programLocationState = mappedKVPs.get("programLocationState").get,
+      programLocationCountry = mappedKVPs.get("programLocationCountry").get,
       pi = mappedKVPs.get("pi").get,
+      nonProfitStatus = mappedKVPs.get("nonProfitStatus").get,
+      billingAccountName = mappedKVPs.get("billingAccountName") match {
+        case Some(x) => Some(x)
+        case _ => None
+      },
```

Comment:
Does this `match` do anything?  Looks like it's effectively an identity operation, unless I'm missing something.